oarepo_communities/cli/__init__.py

```diff
@@ -1,27 +1,40 @@
+#
+# Copyright (c) 2025 CESNET z.s.p.o.
+#
+# This file is a part of oarepo-communities (see https://github.com/oarepo/oarepo-communities).
+#
+# oarepo-communities is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+#
+"""CLI commands for communities."""
+
+from __future__ import annotations
+
 import sys
+from typing import TYPE_CHECKING, cast
 
 import click
 import yaml
 from flask.cli import with_appcontext
 from invenio_access.permissions import system_identity
 from invenio_accounts.models import User
+from invenio_communities.cli import communities
 from invenio_communities.communities.records.api import Community
 from invenio_records_resources.proxies import current_service_registry
-from oarepo_runtime.cli.base import oarepo
-
 
-@oarepo.group()
-def communities():
-    """OARepo communities commands."""
+if TYPE_CHECKING:
+    from invenio_communities.communities.services import CommunityService
+    from invenio_communities.members.services import MemberService
 
 
 @communities.command(name="create")
 @click.argument("slug")
 @click.argument("title")
 @click.option("--public/--private", default=True)
 @with_appcontext
-def create_community(slug, title, public):
-    community_service = current_service_registry.get("communities")
+def create_community(slug: str, title: str, public: bool) -> None:
+    """Create a community."""
+    community_service = cast("CommunityService", current_service_registry.get("communities"))
```

Comment:
maybe create a current_community_service proxy so that this line is not repeated ?